packages/admin-client/src/common/components/CustomDatagridRow/CustomDatagridRow.tsx

```diff
@@ -150,6 +154,7 @@ const DatagridRow: FC<any> = React.forwardRef((props, ref) => {
                                 checked={selectable && selected}
                                 onClick={handleToggleSelection}
                                 disabled={!selectable}
+                                sx={{ alignItems: "start", padding: "6px 9px" }}
```

Comment:
Can you try to refactor this to use relative sizings instead of pixels? Either via `em` or the built-in react sizings (i.e. just using a number).

Reply:
Fixed.